website/integrations/services/sharepoint-se/index.md

```diff
@@ -0,0 +1,379 @@
+-   Resolve user's membership
+
+:::caution
+[LDAPCP](https://www.ldapcp.com/docs/overview/introduction/) must be installed on the target SharePoint Farm.
+:::
+
+### Step 1: Assign LDAPCP as Claim provider for the Identity Token Issuer
+
+The following PowerShell script must be updated according to your environment and executed as **Farm Admin account** with **elevated privileges** on a SharePoint Server.
+
+:::caution
+
+-   Update placeholders
+-   Read all script's comments.
+
+:::
+
+```PowerShell
+Add-PSSnapin microsoft.sharepoint.powershell
+$trustedTokenIssuerName = "sp.issuerName"
+
+$sptrust = Get-SPTrustedIdentityTokenIssuer $trustedTokenIssuerName
+$sptrust.ClaimProviderName = "LDAPCP"
+$sptrust.Update()
+```
+
+### Step 2: Configure LDAPCP Claim types
+
+From the SharePoint Central Administration opened as a Farm Administrator:
+
+1. Open "**Security > LDAPCP Configuration > Claim types configuration**" page
+2. Update the mapping table as the following.
+
+| Claim type                                                    | Entity type | LDAP class | LDAP Attribute to query | LDAP attribute to display | PickerEntity metadata |
+| ------------------------------------------------------------- | ----------- | ---------- | ----------------------- | ------------------------- | --------------------- |
+| http://schemas.microsoft.com/identity/claims/objectidentifier | User        | user       | uid                     | sn                        | UserId                |
+| LDAP attribute linked to the main mapping for object User     | User        | user       | mail                    |                           | Email                 |
+| LDAP attribute linked to the main mapping for object User     | User        | user       | sn                      |                           | DisplayName           |
+| http://schemas.microsoft.com/ws/2008/06/identity/claims/role  | Group       | group      | cn                      |                           | DisplayName           |
+| LDAP attribute linked to the main mapping for object Group    | Group       | group      | uid                     |                           | SPGroupID             |
+
+### Step 3: Create an Authentik LDAP Outpost
+
+From the Authentik Admin Dashboard:
+
+:::note
+The following procedure apply to an Authentik deployment within Kubernetes.
+
+For other kind of deployment, please refer to the Authentik documentation.
+:::
+
+1. Follow Authentik [LDAP Provider Generic Setup](https://version-2023-10.goauthentik.io/docs/providers/ldap/generic_setup) with the following steps :
+    - **Create User/Group** to create a "service account" for `ldap.outpostServiceAccount` and a searchable group of users & groups
+    - **LDAP Flow** to create the authentication flow for the LDAP Provider
+    - **LDAP Provider** to create an LDAP provider which can be consumed by the LDAP Application
+    - **LDAP Application** to create the application being used by the LDAP Outpost
```

Comment:
Could we instead select the provider as a backchannel one in the previously-created Oauth application?

Reply:
(cc. @BeryJu)

Reply:
It works if you upgrade to 2023.10.3 because of the back channel selector bug.

Reply:
Afterwards, it depends mainly on requirements related to auth flows / policies on the ldap provider.

In my lab case, the backchannel works fine, however within an enterprise network may be not.

Reply:
Yeah, that's fine to include even if it's only possible with 2023.10.3.